src/test/java/seedu/address/logic/commands/FindCommandTest.java

```diff
@@ -1,63 +1,95 @@
 import static seedu.address.commons.core.Messages.MESSAGE_PERSONS_LISTED_OVERVIEW;
 import static seedu.address.logic.commands.CommandTestUtil.assertCommandSuccess;
+import static seedu.address.testutil.TypicalPersons.ALBERT;
+import static seedu.address.testutil.TypicalPersons.ANGMEIHUA;
+import static seedu.address.testutil.TypicalPersons.BART;
 import static seedu.address.testutil.TypicalPersons.CLARK;
 import static seedu.address.testutil.TypicalPersons.EDWARD;
 import static seedu.address.testutil.TypicalPersons.FORD;
+import static seedu.address.testutil.TypicalPersons.ISAAC;
+import static seedu.address.testutil.TypicalPersons.JOOKOON;
+import static seedu.address.testutil.TypicalPersons.KEVIN;
+import static seedu.address.testutil.TypicalPersons.LAVENDER;
 import static seedu.address.testutil.TypicalPersons.getTypicalEduMate;
 
 import java.util.Arrays;
 import java.util.Collections;
+import java.util.List;
 
 import org.junit.jupiter.api.Test;
 
+import seedu.address.logic.parser.CliSyntax;
+import seedu.address.logic.parser.Prefix;
 import seedu.address.model.Model;
 import seedu.address.model.ModelManager;
 import seedu.address.model.UserPrefs;
-import seedu.address.model.person.NameContainsKeywordsPredicate;
+import seedu.address.model.person.ContainsKeywordsPredicate;
+import seedu.address.model.person.Person;
 
 /**
  * Contains integration tests (interaction with the Model) for {@code FindCommand}.
  */
 public class FindCommandTest {
-    private Model model = new ModelManager(getTypicalEduMate(), new UserPrefs());
-    private Model expectedModel = new ModelManager(getTypicalEduMate(), new UserPrefs());
+    private final Model model = new ModelManager(getTypicalEduMate(), new UserPrefs());
+    private final Model expectedModel = new ModelManager(getTypicalEduMate(), new UserPrefs());
 
     @Test
     public void equals() {
-        NameContainsKeywordsPredicate firstPredicate =
-                new NameContainsKeywordsPredicate(Collections.singletonList("first"));
-        NameContainsKeywordsPredicate secondPredicate =
-                new NameContainsKeywordsPredicate(Collections.singletonList("second"));
+        List<String> firstKeyword = Collections.singletonList("first");
+        List<String> secondKeyword = Collections.singletonList("second");
+        create_equals(firstKeyword, secondKeyword, CliSyntax.PREFIX_NAME);
+        create_equals(firstKeyword, secondKeyword, CliSyntax.PREFIX_PHONE);
+        create_equals(firstKeyword, secondKeyword, CliSyntax.PREFIX_EMAIL);
+        create_equals(firstKeyword, secondKeyword, CliSyntax.PREFIX_TELEGRAM_HANDLE);
+        create_equals(firstKeyword, secondKeyword, CliSyntax.PREFIX_ADDRESS);
+        create_equals(firstKeyword, secondKeyword, CliSyntax.PREFIX_MODULE_TAG);
+        create_equals(firstKeyword, secondKeyword, CliSyntax.PREFIX_GROUP_TAG);
+    }
+
+    public void create_equals(List<String> firstKeyword, List<String> secondKeyword, Prefix prefix) {
```

Comment:
If it is not a test case, it should not use snake_case.

Reply:
Changed to camel case.